tests/fixtures/__init__.py

```diff
@@ -263,15 +268,14 @@ def find_pipeline_script(commands: Union[list[str], list[list[str]]]) -> str:
         print(command)
         raise Exception("Unable to find a string in the nested command.")
 
-    match = re.search(r"\/pipeline\/([\w\/-]+)\.(py|sh)", command)
-    #                               ^^^^^^^^^^   ^^^^^
-    #                               |            |
-    #    Match the path to the script            Match the extension
+    # $VCS_PATH/taskcluster/scripts/pipeline/train-taskcluster.sh
+    # $VCS_PATH/pipeline/alignment/generate-alignment-and-shortlist.sh
+    match = re.search(r"[\$\w\/]*\/pipeline\/[\w\/-]+\.(py|sh)", command)
```

Comment:
Suggestion: I would prefer to maintain regex documentation on the logical groups of the regex, as this will be easier to maintain. Regexes are a frequent source of bugs and confusion. This is a good candidate for `r"""` multiline regexes with comments.